<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>203</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/images/list.png</normaloff>:/icons/images/list.png</iconset>
        </attribute>
        <attribute name="label">
         <string>列表项操作</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="tBtnListInit">
           <property name="text">
            <string>tBtnLIstInit</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="tBtnListClear">
           <property name="text">
            <string>tBtnListClear</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="tBtnListInsert">
           <property name="text">
            <string>tBtnListInsert</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="tBtnListAppend">
           <property name="text">
            <string>tBtnListAppend</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QToolButton" name="tBtnListDelete">
           <property name="text">
            <string>tBtnListDelete</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>203</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/images/paixu.png</normaloff>:/icons/images/paixu.png</iconset>
        </attribute>
        <attribute name="label">
         <string>列表排序</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="2" colspan="2">
          <widget class="QCheckBox" name="chkBoxSorting">
           <property name="text">
            <string>允许排序</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>升序排序</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::UpArrow</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QToolButton" name="toolButton_7">
           <property name="text">
            <string>降序排序</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::DownArrow</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>88</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>200</width>
          <height>203</height>
         </rect>
        </property>
        <attribute name="label">
         <string>信号发射时机</string>
        </attribute>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>42</y>
           <width>161</width>
           <height>158</height>
          </rect>
         </property>
        </widget>
        <widget class="QToolButton" name="tBtnClearText">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>12</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>清空文本</string>
         </property>
        </widget>
        <widget class="QToolButton" name="tBtnAddLine">
         <property name="geometry">
          <rect>
           <x>82</x>
           <y>12</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>添加空行</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>375</width>
          <height>256</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>当前项变化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editCutItemText"/>
           </item>
           <item>
            <widget class="QCheckBox" name="chkListEditable">
             <property name="text">
              <string>项可编辑</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QToolButton" name="tBtnSelectItem">
             <property name="text">
              <string>选择选项</string>
             </property>
             <property name="popupMode">
              <enum>QToolButton::MenuButtonPopup</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tBtnSelAll">
             <property name="text">
              <string>tBtnSelAll</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tBtnSelNone">
             <property name="text">
              <string>tBtnSelNone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tBtnSelInvs">
             <property name="text">
              <string>tBtnSelInvs</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons/images/paint.png</normaloff>:/icons/images/paint.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item2</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons/images/paint.png</normaloff>:/icons/images/paint.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListInit"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actListInit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/list.png</normaloff>:/icons/images/list.png</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/clear.png</normaloff>:/icons/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/insert.png</normaloff>:/icons/images/insert.png</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入  </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/append.png</normaloff>:/icons/images/append.png</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/delete.png</normaloff>:/icons/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/quit.png</normaloff>:/icons/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSelAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>空选</string>
   </property>
   <property name="toolTip">
    <string>空选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
